Makefile

```diff
@@ -36,14 +36,20 @@ nodelink-controller:
 
 .PHONY: machine-healthcheck
 machine-healthcheck:
-	$(DOCKER_CMD) ./hack/go-build.sh machine-healthcheck 
+	$(DOCKER_CMD) ./hack/go-build.sh machine-healthcheck
 
 .PHONY: build-integration
 build-integration: ## Build integration test binary
 	@echo -e "\033[32mBuilding integration test binary...\033[0m"
 	mkdir -p bin
 	$(DOCKER_CMD) go build $(GOGCFLAGS) -o bin/integration github.com/openshift/machine-api-operator/test/integration
 
+.PHONY: deploy-kubemark
+deploy-kubemark:
+	kustomize build config | kubectl apply -f -
```

Comment:
Is kustomize necessary here?

Reply:
Suggesting to run it without the Makefile?

Reply:
No, I'm suggesting to not use kustomize.  Not sure what it's doing.

Reply:
Just taking all manifests and pumping them into a single stream. Running just one command to deploy all manifests at once.

Reply:
I believe kubectl can do that too; just do `kubectl create -f  ./`

Reply:
The `kustomize` is among other things changing default namespace and patching manifests. Something simple `kubectl` can not do.

Reply:
cluster-api, along with kubebuilder and so operator-sdk use kustomize. I'm in favour of using it here for manifest generation and customization